CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ that users understand how the changes affect the new version.
 -->
 version 3.1.0
 ---------------------------
++ Generalize sort task to now also sort by position, instead of just read name.
```

Comment:
Please also state that the name of the task was changed.

Reply:
Ah yes, will do.